Add `py.typed` marker to satisfy mypy

```
src/hermes/client.py:3: error: Skipping analyzing "darbiadev_shipping": module is installed, but missing library stubs or py.typed marker  [import-untyped]
src/hermes/client.py:3: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "darbiadev-shipping"
-version = "0.5.0"
+version = "0.5.1"
 description = "darbiadev-shipping"
 readme = "README.md"
 authors = [
@@ -43,6 +43,9 @@ docs = [
 [build-system]
 requires = ["setuptools", "wheel"]
 build-backend = "setuptools.build_meta"
+
+[tool.setuptools.package-data]
+darbiadev_shipping = ["py.typed"]
 
 [tool.ruff]
 preview = true

--- a/src/darbiadev_shipping/py.typed
+++ b/src/darbiadev_shipping/py.typed
@@ -1,0 +1,1 @@
+# Marker file for PEP 561.

